account_financial_risk/views/res_partner_view.xml

```diff
@@ -7,22 +7,18 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="priority" eval="20" />
-        <field name="groups_id" eval="[(4, ref('account.group_account_manager'))]" />
```

Comment:
You have to put this group on the view elements instead, not simply remove it.

Reply:
I wonder if this isn't for some debate. We typically remove this group anyway from view, replacing with other groups, because more than an accountant needs to be able to see limits, and risk allow_edit takes care of the rest. Also, in larger companies, not all accountants can edit credit limits, and not everyone who can edit credit limits is an accountant, so we have a completely different security group to do this.

Right now that is a single hidden group, but I could easily make a dedicated Credit Control group with 3 levels say User, Approver, Manager - user is readonly access and the group for the page in this view, approver also readonly here but can approve (currently group_overpass_partner_risk_exception) and manager can approve and set limits. Manager is implied for accountant. What do you think?

Reply:
Once merged, you can improve the module adding more granularity through levels for having this visible as readonly, and another allowing to edit.